Makefile

```diff
@@ -200,7 +200,8 @@ CRDS_CILIUM_V2ALPHA1 := ciliumendpointslices \
                         ciliumbgppeeringpolicies \
                         ciliumloadbalancerippools \
                         ciliumnodeconfigs \
-                        ciliumcidrgroups
+                        ciliumcidrgroups \
+                        ciliumworldcidrsets
```

Comment:
You probably also want to bump `CustomResourceDefinitionSchemaVersion` if we add new CRDs.